presto-tests/src/main/java/com/facebook/presto/tests/QueryAssertions.java

```diff
@@ -147,12 +149,20 @@ public static void assertEqualsIgnoreOrder(Iterable<?> actual, Iterable<?> expec
         ImmutableMultiset<?> actualSet = ImmutableMultiset.copyOf(actual);
         ImmutableMultiset<?> expectedSet = ImmutableMultiset.copyOf(expected);
         if (!actualSet.equals(expectedSet)) {
-            fail(format("%snot equal\nActual %s rows:\n    %s\nExpected %s rows:\n    %s\n",
+            Multiset<?> actualDiff = Multisets.difference(actualSet, expectedSet);
```

Comment:
I would rename `actualDiff` to extra or unexpected rows, and `expectedDiff` to missing rows. 

Also I would use that naming in message. I find it more clear.